rhel7/Dockerfile.pgo-backrest-repo.rhel7

```diff
@@ -3,8 +3,7 @@ ARG BASEVER
 FROM ${PREFIX}/pgo-base:rhel7-${BASEVER}
 
 LABEL name="pgo-backrest-repo" \
-	run='docker run -d -p 8080:80 --name=web-app web-app' \
-	summary="Crunchy Data PostgreSQL Operator - pgo-backrest-repo" \
+	summary="Crunchy PostgreSQL Operator - pgo-backrest-repo" \
```

Comment:
I prefer the `summary` and `description` in the CentOS image, `Crunchy PostgreSQL Operator - pgBackRest Repository`.